resources/qml/ImageButton.qml

```diff
@@ -18,7 +19,7 @@ AbstractButton {
 
         // Workaround, can't get icon.source working for now...
         anchors.fill: parent
-        source: "image://colorimage/" + image + "?" + (button.hovered ? highlightColor : buttonTextColor)
+        source: "image://colorimage/" + image + "?" + ((button.hovered && changeColorOnHover) ? highlightColor : buttonTextColor)
```

Comment:
Okay, there is an issue here. If image is "" or null, it causes this to log a lot of warnings.

Reply:
What would be the best method of fixing this? Perhaps just setting `source` to "" if `image == ""`?

Reply:
I guess that this probably was a problem before as well.

Reply:
> What would be the best method of fixing this? Perhaps just setting `source` to "" if `image == ""`?

Yeah, probably. This was not an issue before, since there was always an image before.

Reply:
True. 

> setting `source` to "" if `image == ""`
This works.